### ISSUE TITLE
Redeploying with mupx leads to node-fiber missing error

### ISSUE THREAD

**Issue by walleXD** [arunoda/meteor-up-legacy#929]
After I have made changes to my project and redeploy using the `mupx deploy` command it seems that the after the redeployment my app fails to start and i get the followign error:
`Error:`/bundle/bundle/programs/server/node_modules/fibers/bin/linux-x64-v8-3.14/fibers.node`is missing. Try reinstalling`node-fibers`?
`

Any ideas how to fix it?

I am on meteor 1.3-rc releases

Thanks in advance! 

Also, i tried to deloy using node 0.10.x but it didn't help. It was mentioned by @arunoda in this [issue](https://github.com/arunoda/meteor-up/issues/342)

**Comment by ramzwatcher** [comment]
same issue with me

**Comment by programthis** [comment]
@walleXD @ramzwatcher I got this error after upgrading to Meteor 1.3 but ran mupx setup again before mupx deploy and it seemed to fix the issue.

**Comment by walleXD** [comment]
@programthis The problem still persists for me, even after running `mupx setup` before redeployment. My errors:

`/bundle/bundle/programs/server/node_modules/fibers/fibers.js:16
Throw new Error('`'+ modPath+ '.node`is missing. Try reinstalling`node-fiber
Error: `/bundle/bundle/programs/server/node_modules/fibers/bin/linux-x64-v8-3.14/fibers.node` is missing. Try reinstalling `node-fibers`?
`

Guys, any idea how to fix this? I pretty much am spinning up new Digital ocean droplets everytime I am making changes to my code and I am not sure how to solve this......

Your help would be much appreciated!

**Comment by mattiLeBlanc** [comment]
I am having the same issue:

```
2016-04-01T01:24:49.122459700Z /bundle/bundle/programs/server/node_modules/fibers/fibers.js:16
2016-04-01T01:24:49.122799400Z  throw new Error('`'+ modPath+ '.node` is missing. Try reinstalling `node-fibe
```

waitTime is set to 60, to no avail.

Update: when I stopped the service first and then did mupx deploy it DID work.
SO...

```
mupx stop
mupx deploy
...beer...
```

update 2: 
errm... it seemed to be a fluke. Next time we tried we got the same error.
How is this possible?

**Comment by d-i-b** [comment]
I got the same message. Meteor up is cool but just check the issues, open issues twice closed ones.
Anyway, I managed to get the app running.
ssh into your server first, then reinstall fibers `cd /opt/yourapp/current/bundle/programs/server/node_modules/ ; rm -rf fibers && npm install fibers`
now quit ssh session and try `mupx start`
The bundle should be built on the target server(why not?).

Other than this issue, for now there are 487 issues open and 46 pull requests including  #942 update node version, #773 add system architecture to build(which may fix this issue and bcrypt ones) and #736 introduce mongo dump and restore with many other updates.

**Comment by MasterJames** [comment]
You are correct! I think it now says this is officially depreciated as the port of mupx over at kadirahq is where all efforts and momentum moving forward are focused. Still it's under development and the development branch here called mupx is suppose to be the "stable" version but with changes to Meteor etc going on its probably wise to try the new kadirahq version as you'll also get better official support and PRs are surely welcome.

**Comment by frigginglorious** [comment]
I was having the same issue. I fixed it by by ssh-ing into the vagrant-spk box, and 
sudo apt-get install g++.

For some reason the fibers package needed to be built?

Sandstorm doessn't like to load the grain now, but at least its building.

**Comment by tmorisita** [comment]
I got the same issue to use mup after updating meteor.
I tried to downgrade the version of meteor and node, then I succeeded to deploy to AWS. For the deploying, I had created a new instance Ubuntu 14.04 t1.micro in AWS.

In my project folder, run `meteor update --release 1.3.1`
Edit mup.json in the project like `{ "nodeVersion": "0.10.41", ... }` 
My local environment is Ubuntu 14.04.

I have not tried mupx.

**Comment by MasterJames** [comment]
Yes mupx is the newer version and both here are now pretty much depreciated by the port of a similar docker centric approach to mupx now at kadirahq/meteor-up 
Moving forward with meteor 1.3.2.2/4 you'll want to be using that newer version.

**Comment by idanwe** [comment]
Set `"deployCheckWaitTime"` to `60` fix the problem as mentioned here https://github.com/kadirahq/meteor-up#faq
